.travis/linux.win32.install.sh

```diff
@@ -1,16 +1,30 @@
 #!/usr/bin/env bash
+set -e
 
-PACKAGES="nsis cloog-isl libmpc3 qt4-linguist-tools mingw32
-	mingw32-x-sdl mingw32-x-libvorbis mingw32-x-fluidsynth mingw32-x-stk
+if [ $QT5 ]; then
+	MINGW_PACKAGES="mingw32-x-qt5base"
+else
+	MINGW_PACKAGES="mingw32-x-qt"
+fi
+
+MINGW_PACKAGES="mingw32-x-sdl mingw32-x-libvorbis mingw32-x-fluidsynth mingw32-x-stk
 	mingw32-x-glib2 mingw32-x-portaudio mingw32-x-libsndfile mingw32-x-fftw
 	mingw32-x-flac mingw32-x-fltk mingw32-x-libsamplerate
 	mingw32-x-pkgconfig mingw32-x-binutils mingw32-x-gcc mingw32-x-runtime
-	mingw32-x-libgig mingw32-x-libsoundio"
+	mingw32-x-libgig mingw32-x-libsoundio $MINGW_PACKAGES"
 
-if [ $QT5 ]; then
-	PACKAGES="$PACKAGES mingw32-x-qt5base"
-else
-	PACKAGES="$PACKAGES mingw32-x-qt"
-fi
+export MINGW_PACKAGES
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+. $DIR/linux.win.download.sh win32
+
+PACKAGES="nsis cloog-isl libmpc3 qt4-linguist-tools mingw32 $MINGW_PACKAGES"
 
 sudo apt-get install -y $PACKAGES
+
+# ccache 3.2 is needed because mingw32-x-gcc is version 4.9, which causes cmake
+# to use @file command line passing, which in turn ccache 3.1.9 doesn't support
+pushd /tmp
+wget http://archive.ubuntu.com/ubuntu/pool/main/c/ccache/ccache_3.2.3-1_amd64.deb
+sudo dpkg -i ccache_3.2.3-1_amd64.deb
```

Comment:
This is really the only argument, that we're bypassing package validation by installing directly, rather than from `apt`.  There are some utilities to validate signatures on packages but I didn't have luck on 12.04 with any of them... Perhaps 14.04 will be better... 

 * `dpkg-sig --verify foo.deb` -- Did nothing
 * `debsig-verify` -- Dependency conflicts, couldn't install